mesh_map/src/mesh_map.cpp

```diff
@@ -51,60 +50,86 @@
 #include <lvr2/io/hdf5/MeshIO.hpp>
 #include <mesh_map/mesh_map.h>
 #include <mesh_map/util.h>
-#include <mesh_msgs/MeshGeometryStamped.h>
+#include <mesh_msgs/msg/mesh_geometry_stamped.hpp>
 #include <mesh_msgs_conversions/conversions.h>
 #include <mutex>
-#include <ros/ros.h>
-#include <visualization_msgs/Marker.h>
+#include <rclcpp/rclcpp.hpp>
+#include <visualization_msgs/msg/marker.hpp>
 
 namespace mesh_map
 {
 using HDF5MeshIO = lvr2::Hdf5IO<lvr2::hdf5features::ArrayIO, lvr2::hdf5features::ChannelIO,
                                 lvr2::hdf5features::VariantChannelIO, lvr2::hdf5features::MeshIO>;
 
-MeshMap::MeshMap(tf2_ros::Buffer& tf_listener)
+MeshMap::MeshMap(tf2_ros::Buffer& tf, const rclcpp::Node::SharedPtr& node)
   : tf_buffer(tf_buffer)
-  , private_nh("~/mesh_map/")
```

Comment:
Please note this change.
Afaik, something like a "sub node" handle does not exist in ROS2. In ROS1, this code would nicely put parameters and subscribers into a sub namespace. I tried to replicated the behavior via a constant variable `MESH_MAP_NAMESPACE = "mesh_map"` and add this as a prefix to all params / topics (e.g. see `declare_parameter()` below). I have seen some ROS2 code elsewhere, where subnamespaces were done via `.` instead of `/`. We'll have to see whether this works when we are ready to run some thorough tests.